stage1/init/init.go

```diff
@@ -135,6 +135,7 @@ func parseFlags() *stage1commontypes.RuntimePod {
 	})
 	flag.Var(dnsConfMode, "dns-conf-mode", "DNS config file modes")
 	flag.StringVar(&rp.IPCMode, "ipc", "", "IPC mode --ipc=[auto|private|parent]")
+	flag.StringVar(&rp.KubernetesLogDir, "kubernetes-log-dir", "", "Mount the kubernetes log directory to write CRI logs")
```

Comment:
This is a bit heavy and needs to be put in the stage1 interface protocol. Can't we instead just generate a volume+mount pair in stage0 at `app_add` time? If not, then an annotation on the pod is probably fine for the time being.